### PR TITLE
Upgrade to newer basepom

### PR DESCRIPTION
**Pull request by jhaber**
This upgrades to new basepom and also builds on Java 11

@jboulter @boulter

### CODE REVIEW

File: src/test/java/com/hubspot/jinjava/objects/date/StrftimeFormatterTest.java [hunk 1]
```diff
@@ -94,6 +94,6 @@ public void itConvertsNominativeFormats() {
     ZonedDateTime zonedDateTime = ZonedDateTime.parse("2019-06-06T14:22:00.000+00:00");
 
     assertThat(StrftimeFormatter.formatter("%OB").withLocale(Locale.forLanguageTag("ru")).format(zonedDateTime))
-        .isEqualTo("Июнь");
+        .isIn("Июнь", "июнь");
```
Comment (jhaber) [inline src/test/java/com/hubspot/jinjava/objects/date/StrftimeFormatterTest.java]: Seems like there was a minor change with Java 11; I made the test accept either format so it still works on Java 8

Comment (mattcoley) [reply src/test/java/com/hubspot/jinjava/objects/date/StrftimeFormatterTest.java]: `июнь` is correct, it was a bug with Java 8.

**Comment by boulter** [comment]
Will it still target Java 8 bytecode?

**Comment by jhaber** [comment]
Yup

**Pull request merged**